pkg/controller/nodeipam/OWNERS

```diff
@@ -0,0 +1,9 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+- bowei
```

Comment:
Can we get a second approver? Dislike having a SPF.

Reply:
Would you have any suggestions on who could be added here? cc @bowei